### PR TITLE
completed css-two-column-layout - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-04-02 at 3 52 45 PM

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+.text-body {
+  font-family: Karla, sans-serif;
+  line-height: 150%;
+}
+
+.row {
+  font-size: 14px;
+  display: flex;
+  margin: 0 auto;
+  flex-basis: 50%;
```
Comment (shawnkost) [inline css-two-column-layout/styles.css]: I don't believe this is doing anything. `flex-basis` is a child property, so you'd only apply it to flex items, not a flex parent. So you'd apply it to the columns.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just one change here Bryan!

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (bbrown949) [reply css-two-column-layout/styles.css]: updated.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**